src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -2317,16 +2317,139 @@ image::TenantSPUsers.png[Scheduler Portal for "users" tenant]
 .Scheduler Portal for "Experts" tenant
 image::TenantSPExperts.png[Scheduler Portal for "experts" tenant]
 
-.Scheduler Portal for Admin users
+.Scheduler Portal for Global Administrator
 image::TenantSPAdmins.png[Admins Tenant View]
 
-===== Tenants Node Source Isolation
+===== Tenants Resource Manager Isolation
+
+Apart from scheduler job isolation, it is also possible to isolate the <<_glossary_resource_manager,Resource Manager>> and <<_glossary_node_source,Node Sources>> across tenants.
+
+By default, the Resource Manager displays all Nodes and Node Sources, regardless of tenants.
+Tenant isolation allows to restrict the Nodes and Node Souces that will be visible on the Resource Manager portal.
+
+In this scenario, the Resource Manager may contain:
+
+ * Node Sources that belong to a single tenant and thus only visible to members of this tenant.
+ * Node Sources that don't belong to any tenant and thus visible to all users, regardless of tenant membership.
+
+Statistics - such as number of nodes and nodes history - displayed in the Resource Manager portal will match the user's visibility access.
+
+In order to enable Resource Manager tenants isolation, the following properties must be configured in `PROACTIVE_HOME/config/rm/settings.ini`:
+----
+pa.rm.tenant.filter=true
+pa.rm.jmx.tenant.names=tenant1,tenant2
+----
+
+`pa.rm.jmx.tenant.names` contains the list of tenants names, this setting is mandatory to allow Resource Manager statistics isolation.
+
+When the above properties are configured, the following behavior applies:
+
+ * A Node Source created by an administrator who belongs to a given tenant will be visible only to members of the same tenants. This Node Source will contain Nodes that are restricted to a single tenant.
+ * A Node Source created by an administrator who does not belong to any tenant will be visible to all users, regardless of tenants. This Node Source will contain Nodes that are shared across tenants.
+
+From a given user point of view, the following applies:
+
+ * A user who does not belong to any tenant will only see shared Node Sources.
+ * A user who belongs to a tenant will see shared Node Sources and Node Sources associated with the same tenant.
+ * A user or administrator who has the permission `org.ow2.proactive.scheduler.permissions.TenantAllAccessPermission` will see all Node Sources.
+
+In order to create a Node Source restricted to a tenant, it is thus necessary to create a Resource Manager administrator group that is attached to this tenant.
```

Comment:
> ... , it is thus necessary to create

, it is necessary to create
Thus & therefore  don't serve any argumenting situation here that requires a conjunctive

Reply:
Only used it once or twice 2 years ago 😄 but yeah, that 's what we spend the most time on when doing a PhD, grammar 😆 They should rename it to Gd instead ...